__mocks__/applicationinsights.js

```diff
@@ -1,6 +1,10 @@
-export const appInsights = {
+const appInsights = {
   defaultClient: {
     trackEvent: jest.fn(),
     trackException: jest.fn()
-  }
+  },
+  setup: function () { return this },
+  start: jest.fn()
 }
+
+export default appInsights
```

Comment:
Similarly for this, do we need to go back to a default export here rather than explicitly named one?

Reply:
In this case it is because we are mocking appInsights which uses a default export and is imported as `import appInsights from 'applicationinsights'` in the files we want to mock for.